editor/js/reaction.js

```diff
@@ -462,6 +478,10 @@ function initSelector(node) {
     if (options[i] == 'UNSPECIFIED') {
       option.attr('selected', 'selected');
     }
+    // Don't let users directly select/enter RDKIT_BINARY identifiers.
+    if (options[i] == 'RDKIT_BINARY') {
```

Comment:
I assume this will go away after #247 is fixed?

Reply:
Yes; #247 is slightly more involved, but this line would be removed